src/main/java/filter/expression/FilterExpression.java

```diff
@@ -28,6 +28,11 @@
 
     void applyTo(TurboIssue issue, IModel model) throws QualifierApplicationException;
 
+    // Walks the syntax tree to get all possible filter error but is not severe
+    // enough to cause an error.
```

Comment:
Maybe we can be more specific about what warnings are for. Problems with inputs not severe enough to cause an error?